### ISSUE TITLE
Segfault during startup

### ISSUE THREAD

**Issue by sumpfralle** [freeorion/freeorion#2657]
### Environment

* **FreeOrion Version**: commit a907c0b88
* **Operating System**: Debian testing (i386; 32 Bit)
* **Graphic card used**:
* **Fetched as**
  * Compiled from source

### Description
During statup the game window pops up for a moment before segfaulting.

### Expected Result
Successful start of the game.

### Steps to reproduce
1. `cmake build`
2. `make -C build`
3. `cd build; ./freeorion`

### Details
`gdb` outputs the following:
```
Thread 14 "freeorion" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x9a273b40 (LWP 2811)]
0x00582fd0 in std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >& std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_replace_dispatch<std::_Deque_iterator<char, char&, char*> >(__gnu_cxx::__normal_iterator<char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, __gnu_cxx::__normal_iterator<char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::_Deque_iterator<char, char&, char*>, std::_Deque_iterator<char, char&, char*>, std::__false_type) ()
```

**Comment by o01eg** [comment]
1. Can you print output of `bt` from `gdb`?
2. Could you try with less AI player?

**Comment by o01eg** [comment]
@sumpfralle Could you also configure it with `-DCMAKE_BUILD_TYPE=RelWithDebInfo` ?

**Comment by sumpfralle** [comment]
Thank you for your quick response!

> Could you try with less AI player?

Probably I was not clear enough: the segfault happened right after I started the executable. There was just a flicker of the window visible for a second, before the segfault crashed the program.

Sadly I failed to reproduce the broken build now :(

(with or without `-DCMAKE_BUILD_TYPE=RelWithDebInfo`)

I am very confused and I can hardly imagine, that the build result could be random.

Anyway: thank you for your time and happy hacking!
